### DISCUSSION TITLE
家里的wifi无法使用，申请退款

### DISCUSSION THREAD

**Discussion by 407698432** [getlantern/lantern-forum-cn#451] [General]
反馈链接：https://github.com/getlantern/lantern-forum-cn/discussions/77

已经反馈过好多次，彻底用不了了

账号：4 0 7 6 9 8 4 3 2 @ q q . c o m

**Comment by 407698432** [comment]
![640 (1)](https://github.com/getlantern/lantern-forum-cn/assets/157622338/d5b5dc26-92a8-4b01-8f57-37aed041956a)

**Comment by 407698432** [comment]
![100m1g000001hc1vlDAFC](https://github.com/getlantern/lantern-forum-cn/assets/157622338/fd6be4bb-cbf3-44b7-9988-3b56992606ae)

**Comment by 407698432** [comment]
申请申请退款

**Reply by MaxMKMax** [reply]
@407698432 嗯嗯好的. 帮您提交了!

**Reply by 407698432** [reply]
请求催办

**Reply by 407698432** [reply]
请求催办

**Reply by MaxMKMax** [reply]
![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/abecc8a7-5206-4093-896b-cdbca8225aa8)

@407698432 您好. 您的专业版账号已经使用太久了, 系统已经无法进行退款. 非常抱歉. 对不起我们尽力了!

**Reply by 407698432** [reply]
能否申请只退一半或者三分之一，真的用不了

**Reply by MaxMKMax** [reply]
@407698432 您好. 其实我们在购买协议上有写明是无法退款的. 但由于我们考虑到用户, 我们特别开启退款给某些符合条件的用户. 但您的情况真的是已经用太久了, 系统完全无法操作退款.

我们也想帮忙但系统不允许了. 希望您能理解, 非常抱歉! 我们尽力了. 谢谢! 您要不再试试看最新版本. 您可以检查一下您是否用这 8.0.0 版本? 这个版本应该没什么问题了.
